tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+minversion = 3.8.0
+envlist = py{38,39,310},black,flake8,isort
+isolated_build = true
+skip_missing_interpreters = true
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}
+skip_install = true
+sitepackages = false
+whitelist_externals = poetry
+commands =
+    pip3 install --upgrade poetry
+    poetry install
+    mdapi --version
+    pytest -vvv tests/
+
+[testenv:black]
+commands =
+    pip3 -q install black
```

Comment:
you can use the `deps` key instead (same goes for flake8 and isort)

Reply:
I will look into it. Thanks for marking it out.

Reply:
Fixed with https://github.com/t0xic0der/mdapi/pull/36/commits/2efee7bc95fb6b21c2386b43f62eddf26c1a6b56